google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerHelper.java

```diff
@@ -32,14 +33,11 @@
   public ApplicationServerInfo getApplicationServerInfo(
       ApplicationServerPersistentData persistentData)
       throws CantFindApplicationServerJarsException {
-    final AppEngineSdk sdk = ((AppEngineServerData) persistentData).getSdk();
-    String version = sdk.getVersion();
-    return new ApplicationServerInfo(sdk.getLibraries(),
-        "AppEngine Dev" + (version != null ? " " + version : ""));
+    return new ApplicationServerInfo(new File[]{}, "Cloud SDK");
```

Comment:
Assuming this is the place where the local run config name (visible in the UI) is set, I'm more in favor of calling it something like App Engine Local Server or something like that. Users don't really need to care too much about the Cloud SDK (aside from initially giving the path). Plus Cloud SDK isn't clear to me that its about running a local dev server.